modules/switches.py

```diff
@@ -0,0 +1,107 @@
+import random
+import modules
+import enum
+
+class Switches(modules.Module):
+    class D():
+        UP=False
+        DOWN=True
+    identifiers = ['switches']
+    display_name = "Switches"
+    manual_name = "Switches"
+    help_text = "`{cmd} flip 1 5 3 4 2 5 2` or `{cmd} flip 1534252`. Partial solutions allowed."
+    module_score = 4
+
+    # each number is a switch in the up state
+    invalid_positions = ["3","245","2345","14","145","1345","12","124","123","1234"]
+    positions_for_svg_generation = [[50.5, 174], [65.5, 174], [73, 234], [43, 234]]
```

Comment:
I feel that a tuple fits better for representing coordinates:

```suggestion
    switch_polygon = [(50.5, 174), (65.5, 174), (73, 234), (43, 234)]
```